docs/usage.rst

```diff
@@ -80,6 +79,29 @@ Every contract will always have the following key/value pairs:
 
 Each contract type defines additional options that you supply here.
 
+TOML configuration
```

Comment:
There is INI-style configuration throughout the docs (e.g. on the Contract Types page), not just on this page. I wonder if TOML users coming to those pages might be confused.

Perhaps it would be worth moving this content to a dedicated top-level page called 'TOML support' which could begin with something like:

"While all the examples are in INI format, Import Linter also supports TOML."

My hope would be that the top level menu item would make it more discoverable for those users.

Also open to other ideas.

Reply:
done